tests/test_wave_01.py

```diff
@@ -183,7 +185,8 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    #assert len(updated_data) ["watchedlist"] ==
+    assert movie_to_watch == HORROR_1
```

Comment:
This confirms that the data we input into the test `movie_to_watch` holds `HORROR_1`, but we aren't looking inside the lists that were changed to confirm the elements there are the movie dictionaries that we expect. How could we inspect the contents of the `updated_data["watchlist"]` and `updated_data["watched"]` lists?